openshift-ci/README.md

```diff
@@ -0,0 +1,37 @@
+# MetalLB test env
+
+This scripts installs MetalLB into OCP, for test environment.
+
+## Prerequisites
+
+- Succeed to deploy OCP by dev-scripts (https://github.com/openshift-metal3/dev-scripts)
+- IPv4 is enabled (Currently we targets IPv4. v4v6 case, we only support IPv4 for now)
+
+## Quickstart
+
+Make sure that OCP is deployed by dev-scripts.
+
+To configure MetalLB clone this repo to dev-scripts and run:
+
+```
+$ cd <dev-scripts>/metallb/openshift-ci/
```

Comment:
This is not true anymore I guess

Reply:
We still depend on dev-scripts. We clone metallb to dev-scripts, done here in the CI: https://github.com/openshift/release/pull/25906/files#diff-b105aad5b8eaf216cb20978e8888a34ff7a15bc7e128aec50505a2249ee1138aR29